guide/14-deep-learning/pixel_based_classification.ipynb

```diff
@@ -79,14 +79,20 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>Pre process training&nbsp;data that will be used for training data export</blockquote><p>I think we need to remove the space in Pre process</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>